pcstac/requirements.txt

```diff
@@ -1,23 +1,31 @@
+# pccommon deps
+fastapi==0.67.*
+opencensus-ext-azure==1.0.8
+opencensus-ext-logging==0.1.0
+
+# server deps
 uvicorn[standard]==0.13.3
 uvloop==0.14.0
 gunicorn==20.1.0
 
 orjson==3.5.2
 cachetools==4.2.1
 
-# Installed from source in Dockerfile
-# Including upgrade for pgStac 0.4.0
+# TODO: remove. stac_fastapi Installed from source
+# Including upgrade for pgStac 0.4.3
 # Remove after release >2.2.0
-#
-# stac-fastapi.types==2.2.0
-# stac-fastapi.api==2.2.0
-# stac-fastapi.extensions==2.2.0
-# stac-fastapi.pgstac==2.2.0
+stac_fastapi.types @ git+https://github.com/stac-utils/stac-fastapi.git@d5c9d6cdc4c29588ae4fbe1cf5427a5ce5cde66a#subdirectory=stac_fastapi/types
+stac_fastapi.api @ git+https://github.com/stac-utils/stac-fastapi.git@d5c9d6cdc4c29588ae4fbe1cf5427a5ce5cde66a#subdirectory=stac_fastapi/api
+stac_fastapi.extensions @ git+https://github.com/stac-utils/stac-fastapi.git@d5c9d6cdc4c29588ae4fbe1cf5427a5ce5cde66a#subdirectory=stac_fastapi/extensions
+stac_fastapi.pgstac @ git+https://github.com/stac-utils/stac-fastapi.git@d5c9d6cdc4c29588ae4fbe1cf5427a5ce5cde66a#subdirectory=stac_fastapi/pgstac
+# "stac-fastapi.types==2.2.0",
+# "stac-fastapi.api==2.2.0",
+# "stac-fastapi.extensions==2.2.0",
+# "stac-fastapi.pgstac==2.2.0",
 
-packaging==20.8
-fastapi==0.67.*
-opencensus-ext-azure==1.0.8
-opencensus-ext-logging==0.1.0
+pystac==1.*
+
+# TODO: remove, pypgstac is not really needed to run stac-fastapi application
 pypgstac==0.4.0
```

Comment:
Looks like this ends up being the version used for migrations, rather than the version coming from stac-fastapi. Could you bump this to 0.4.3 to match for the local dev env?